packages/openapi-ts/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@hey-api/openapi-ts",
-  "version": "0.66.3",
+  "version": "0.67.0",
```

Comment:
Can you revert this change? Not sure how that happened

Reply:
reverted